public/index.html

```diff
@@ -24,7 +24,11 @@
       work correctly both with client-side routing and a non-root public URL.
       Learn how to configure a non-root public URL by running `npm run build`.
     -->
-    <title>React App</title>
```

Comment:
Can i understand that the package bootstrap is already installed and called in index.js 

import "bootstrap/dist/css/bootstrap.min.css";

It is called this way, but you still added duplicates calls for the bootstrap css in the public/index.html folder .  

Can we have no duplicate here and consistency